src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.char.cs

```diff
@@ -133,108 +133,6 @@ public void ReadOnlyBufferSlice_ChecksEnd()
             Assert.Throws<ArgumentOutOfRangeException>(() => buffer.Slice(70, buffer.Start));
         }
 
-        [Fact]
-        public void SegmentStartIsConsideredInBoundsCheck()
```

Comment:
The common test only contains the test for T = byte, but this is testing T = char (https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.Common.cs#L18).

I think we should keep the tests in general. No need to remove tests at this point.